client/reader/sidebar/lists/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Just a reminder that by adding the `lists` folder you are creating a "component" namespace, so it should probably be `reader-lists` or `reader-sidebar-lists`.